pkg/process/config/config.go

```diff
@@ -371,6 +374,19 @@ func NewAgentConfig(loggerName config.LoggerName, yamlPath, netYamlPath string)
 	return cfg, nil
 }
 
+// RegisterRuntimeSettings registers settings to be added to the runtime config.
```

Comment:
nit: not sure if this is leftover refactor that didn't update the comment with the new function name.
```suggestion
// initRuntimeSettings registers settings to be added to the runtime config.
```